cargo-generate.toml

```diff
@@ -0,0 +1,17 @@
+# See docs @ https://cargo-generate.github.io/cargo-generate/
+[template]
+cargo_generate_version = ">=0.22"
+
+[placeholders]
+description = { type = "text", prompt = "Long description for the app:", defalut = "A COSMIC application, built using the [`libcosmic` template](https://github.com/pop-os/cosmic-app-template/" }
```

Comment:
defalut -> default

It's also better to describe it as an application for COSMIC, since the former gives the impression that it's an official COSMIC application.

Reply:
```suggestion
description = { type = "text", prompt = "Long description for the app:", default = "An application built for COSMIC, based on the [`libcosmic` template](https://github.com/pop-os/cosmic-app-template/" }
```

Reply:
experimenting a bit, I cannot get this default to trigger at all (`ctrl+d` leaves a newline only), so removing the default